CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ project("Datadog.APM.Native" VERSION 2.18.0)
 # Environment detection
 # ******************************************************
 
+option(UNIVERSAL "Build as Universal binary" OFF)
```

Comment:
Should we default this to ON, for safety? I guess not, as it would be a pain elsewhere? 🤔